exoscale/datasource_exoscale_domain_record.go

```diff
@@ -0,0 +1,196 @@
+package exoscale
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"regexp"
+	"time"
+
+	"github.com/exoscale/egoscale"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func datasourceDomainRecord() *schema.Resource {
+	return &schema.Resource{
+
+		Read: datasourceDomainRecordRead,
+
+		Schema: map[string]*schema.Schema{
+			"domain": {
+				Type:        schema.TypeString,
+				Description: "Domain of the Record",
+				Required:    true,
+			},
+			"filter": {
+				Type:     schema.TypeList,
+				Required: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:          schema.TypeInt,
+							Optional:      true,
+							ConflictsWith: []string{"filter.0.name", "filter.0.record_type", "filter.0.content"},
+						},
+						"name": {
+							Type:          schema.TypeString,
+							Optional:      true,
+							ConflictsWith: []string{"filter.0.id", "filter.0.record_type", "filter.0.content"},
+						},
+						"record_type": {
+							Type:          schema.TypeString,
+							Optional:      true,
+							ConflictsWith: []string{"filter.0.id", "filter.0.content"},
```

Comment:
Isn't this attribute conflicting with `filter.0.name` too, since the opposite does? 🤔

Reply:
I have just forgot to remove conflict attribute  `filter.0.record_type` in `filter.0.name`.
Thank you